.github/workflows/ci.yml

```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches: [master]
```

Comment:
Concern: should we change this to '**'?

Reply:
Probably, we might do some testing or exploring a new feature and having the tests directly on the branch would be easier than creating a PR that may not even be merged.